### PR TITLE
Feature/add sugar

### PR DESCRIPTION
**Pull request by fungjj92**
Let me know if you poke around and find a missing spot or case where the remote dataset title doesn't alias. Part of issue #116

### CODE REVIEW

**Comment by fungjj92** [comment]
Fixing broken tests and a bug.

**Comment by fungjj92** [comment]
Fixed and ready for review! Thanks @CloudNiner for the save.

File: lib/assets/javascripts/cartodb/common/dialogs/delete_items_view.js [hunk 1]
```diff
@@ -144,13 +144,15 @@ module.exports = BaseDialog.extend({
     );
   },
 
-  _getFirstItemName: function() {
+  _setTitle: function() {
```
Comment (sharph) [inline lib/assets/javascripts/cartodb/common/dialogs/delete_items_view.js]: This is a confusing name for this function, as it is a getter and not a setter.

File: lib/assets/javascripts/cartodb/old_common/utils.js [hunk 2]
```diff
@@ -293,3 +293,13 @@
     var value = object[property];
     return _.isFunction(value) ? value.apply(object, Array.prototype.slice.call(arguments, 2)) : value;
   };
+
+
+  /*  Titlecase title function used with aliasing throughout. Various model inputs (vis, table, model)
+      may be appropriate */
+  cdb.Utils.titlecaseName = function(model) {
```
Comment (sharph) [inline lib/assets/javascripts/cartodb/old_common/utils.js]: I think "title case" is two words and that this should be reflected in the camel-case.

**Comment by sharph** [comment]
everything else LGTM.

**Pull request merged**